### PR TITLE
Remove ActiveJob

### PR DESCRIPTION
**Pull request by aub**
This removes ActiveJob in favor of using Sidekiq directly.

### CODE REVIEW

File: app/jobs/disability_claim/document_upload.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
```
Comment (austin-martinez) [inline app/jobs/disability_claim/document_upload.rb]: Should we move these to a `app/workers` directory to be consistent with Sidekiq convention or just leave in `jobs` to match Rails convention?

Comment (aub) [inline app/jobs/disability_claim/document_upload.rb]: Good idea, I'll move them.

File: config/application.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 require 'rails'
 # Pick the frameworks you want:
 require 'active_model/railtie'
-require 'active_job/railtie'
+# require 'active_job/railtie'
```
Comment (carlmjohnson) [inline config/application.rb]: Drop this line?

Comment (aub) [inline config/application.rb]: If you look at that file, all of the other remove Rails components are commented out as well. I think it makes sense to keep it this way so we have a record of which ones we have removed.

**Pull request merged**